internals/cli/cmd_add.go

```diff
@@ -32,12 +32,13 @@ label (or append if the label is not found).
 `
 
 type cmdAdd struct {
-	clientMixin
 	Combine    bool `long:"combine"`
 	Positional struct {
 		Label     string `positional-arg-name:"<label>" required:"1"`
 		LayerPath string `positional-arg-name:"<layer-path>" required:"1"`
 	} `positional-args:"yes"`
+
+	client *client.Client
```

Comment:
super nit: Any reason why you moved this all the way down and introduced a space?

Reply:
In order to "cuddle" (group) the arguments and command members.

Reply:
I'm with Fred here, can we please preserve the position of the client, and indeed have it as the first field of every struct that requires it?  This is the common ground for all commands that take it, so it shoud be early in the definition.